.env.template

```diff
@@ -4,3 +4,7 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+trello_key={ADD YOUR TRELLO KEY HERE}
+trello_token={ADD YOUR TRELLO TOKEN HERE}
+trello_bord_id={ADD YOU TRELLO BOARD ID HERE}
```

Comment:
Should be `trello_bord_id`, you're missing an a in board

Reply:
Will commit this to master